app/code/Magento/Customer/Block/Address/Edit.php

```diff
@@ -240,14 +240,32 @@ public function getBackUrl()
     /**
      * Return the Url for saving.
      *
+     * @param int $defaultShipping Default shipping flag
+     * @param int $defaultBilling Default billing flag
      * @return string
      */
-    public function getSaveUrl()
+    public function getSaveUrl($defaultShipping = 0, $defaultBilling = 0)
```

Comment:
Maintain backward compatibility for the getSaveUrl method by deprecating the old method and introducing a new one